### PR TITLE
Improve Install instructions in README

### PR DESCRIPTION
**Pull request by AliSoftware**
* [x] I've added an entry in the `CHANGELOG.md` file to explain my changes and credit myself
  * [x] Add the entry in the appropriate section (Bug Fix / New Feature / Breaking Change / Internal Change)
  * [x] Add a period and 2 spaces at the end of your short entry description
  * [x] Add links to your GitHub profile to credit yourself and to the related PR and issue after your description

---

## Motivation

I've seen a lot of people preferring downloading the ZIP to install SwiftGen, especially unarchiving it in their project folder and committing it to git, so that every co-worker are assured to use the same version on all their machines for this project. This also allows people to have different versions of SwiftGen for different projects, in case some projects are ready to update but not others.

That's also the installation method that is used by Sourcery, so we'd have some consistency here

### CODE REVIEW

**Comment by Liquidsoul** [comment]
Maybe we could add that the project can be build and installed from source using `brew --HEAD` in the `Compile from source` section?

**Comment by AliSoftware** [comment]
Indeed. Never tried that though so better be sure it works and indeed points to master when we do that, but otherwise yeah

File: README.md [hunk 1]
```diff
@@ -70,6 +106,16 @@ $ rake cli:install
 # Binary will be installed in `~/swiftgen/bin`, framworks in `~/swiftgen/fmk` and templates in `~/swiftgen/tpl`
```
Comment (Liquidsoul) [inline README.md]: We've got a typo here `framworks`, maybe you can fix that in this PR ?

**Comment by Liquidsoul** [comment]
I have just tried it and it works flawlessly 👌

File: README.md [hunk 2]
```diff
@@ -94,20 +94,27 @@ swiftgen …
...
 <summary><strong>Compile from source</strong> <em>(only recommended if you need features from master or want to test a PR)</em></summary>
 
-Alternatively, you can clone the repository and use `rake cli:install` to build the tool.  
-_With this solution you're sure to build and install the latest version from `master` and have access to features which might not have been released yet._
+This solution is when you want to build and install the latest version from `master` and have access to features which might not have been released yet.
+
+* If you have `homebrew` installed, you can use the following command to build and install the latest commit:
+
+```sh
+brew install swiftgen --HEAD
```
Comment (AliSoftware) [inline README.md]: @Liquidsoul that's the exact right command, right?

Comment (Liquidsoul) [inline README.md]: yes 👍

**Pull request merged**